index.rst

```diff
@@ -442,12 +442,12 @@ Programming
     |                         |     f = @(x) a + x                   |        return a + x                   |        f(x) = a + x           |
     |                         |     f(1.0)                           |    f(1.0)                             |        f(1.0)                 |
     +-------------------------+--------------------------------------+---------------------------------------+-------------------------------+
-    |                         | .. code-block:: matlab               | .. code-block:: python                | .. code-block:: julia         |
+    |                         |  No simple syntax to achieve this    | .. code-block:: python                | .. code-block:: julia         |
```

Comment:
Sounds good to me.

the only alternative to consider is  `No consistent or simple syntax to achieve this<https://blogs.mathworks.com/loren/2007/03/22/in-place-operations-on-data/>`___

In some ways, looking at this confusion is a good sales job for python or julia!

Reply:
thanks @jlperla :-)

I'll try and add the link. Table formatting in RST (well any simple markup really) is awful :-)

Reply:
Can't wait for myst :-)  If it takes you more than about 3 minutes it probably isn't worth it.  We should stick with your change.

Reply:
Done